README.md

```diff
@@ -1,74 +1,46 @@
-# InfluxDB Foreign Data Wrapper for PostgreSQL
-This PostgreSQL extension is a Foreign Data Wrapper (FDW) for InfluxDB (version 1.x series).
-
-The current version can work with PostgreSQL 10, 11, 12, 13 and 14.
-
-Go version should be 1.10.4 or later.
-## Installation
-Install InfluxDB Go client library
-<pre>
-go get github.com/influxdata/influxdb1-client/v2
-</pre>
-
-Add a directory of pg_config to PATH and build and install influxdb_fdw.
-<pre>
-make USE_PGXS=1 with_llvm=no
-make install USE_PGXS=1 with_llvm=no
-</pre>
-with_llvm=no is necessary to disable llvm bit code generation when PostgreSQL is configured with --with-llvm because influxdb_fdw use go code and cannot be compiled to llvm bit code.
-
-If you want to build influxdb_fdw in a source tree of PostgreSQL instead, use
-<pre>
-make with_llvm=no
-make install  with_llvm=no
-</pre>
-
-## Usage
-### Load extension
-<pre>
-CREATE EXTENSION influxdb_fdw;
-</pre>
-
-### Create server
-<pre>
-CREATE SERVER influxdb_server FOREIGN DATA WRAPPER influxdb_fdw OPTIONS
-(dbname 'mydb', host 'http://localhost', port '8086') ;
-</pre>
-
-### Create user mapping
-<pre>
-CREATE USER MAPPING FOR CURRENT_USER SERVER influxdb_server OPTIONS(user 'user', password 'pass');
-</pre>
-
-### Create foreign table
-You need to declare a column named "time" to access InfluxDB time column.
-<pre>
-CREATE FOREIGN TABLE t1(time timestamp with time zone , tag1 text, field1 integer) SERVER influxdb_server OPTIONS (table 'measurement1');
-</pre>
-You can use "tags" option to specify tag keys of a foreign table.
-<pre>
-CREATE FOREIGN TABLE t2(tag1 text, field1 integer, tag2 text, field2 integer) SERVER influxdb_server OPTIONS (tags 'tag1, tag2');
-</pre>
-
-### Import foreign schema
-<pre>
-IMPORT FOREIGN SCHEMA public FROM SERVER influxdb_server INTO public;
-</pre>
-
-### Access foreign table
-<pre>
-SELECT * FROM t1;
-</pre>
+InfluxDB Foreign Data Wrapper for PostgreSQL
+============================================
+
+This is a foreign data wrapper (FDW) to connect [PostgreSQL](https://www.postgresql.org/)
+to [InfluxDB](https://www.influxdata.com) database file. This FDW works with PostgreSQL 11, 12, 13, 14, 15 and confirmed with InfluxDB 1.x series.
```

Comment:
The new release including support 2.x series, would you please update it for me?

confirmed with:
  - InfluxDB 1.8: with either [influxdb1-go](#install-influxdb-go-client-library) client or [influxdb-cxx](#install-influxdb_cxx-client-library) client.
  - InfluxDB 2.2: with [influxdb-cxx](#install-influxdb_cxx-client-library) client via InfluxDB v1 compatibility API.

Reply:
Added, thanks for detailed information, @weiting1lee !

Reply:
Thank you.